src/api/config/initializers/influxdb_rails.rb

```diff
@@ -1,9 +1,11 @@
 InfluxDB::Rails.configure do |config|
-  config.influxdb_database = CONFIG['influxdb_database']
-  config.influxdb_username = CONFIG['influxdb_username']
-  config.influxdb_password = CONFIG['influxdb_password']
-  config.influxdb_hosts    = CONFIG['influxdb_hosts'] || [] # default is otherwise localhost
-  config.influxdb_port     = CONFIG['influxdb_port']
-  config.retry             = CONFIG['influxdb_retry']
-  config.use_ssl           = CONFIG['influxdb_ssl']
+  config.influxdb_database   = CONFIG['influxdb_database']
+  config.influxdb_username   = CONFIG['influxdb_username']
+  config.influxdb_password   = CONFIG['influxdb_password']
+  config.influxdb_hosts      = CONFIG['influxdb_hosts'] || [] # default is otherwise localhost
+  config.influxdb_port       = CONFIG['influxdb_port']
+  config.retry               = CONFIG['influxdb_retry']
+  config.use_ssl             = CONFIG['influxdb_ssl']
+  config.time_precision      = CONFIG['influxdb_time_precision']
+  config.series_name_for_sql = CONFIG['influxdb_series_name_for_sql']
```

Comment:
Why does this have to be configurable?

Reply:
And what about defaults for all of those?

Reply:
All other ones have defaults except ``influxdb_series_name_for_sql`` because it is disabled by default.

Reply:
k